ui/components/Page.tsx

```diff
@@ -17,7 +17,7 @@ export type PageProps = {
 };
 
 export const Content = styled(Flex)`
-  background-color: rgba(255, 255, 255, 0.75);
+  background-color: rgba(255, 255, 255, 1);
```

Comment:
💡 If we're ditching the opacity we can use `props.theme.colors.white` since I know there were plans to do a dark mode eventually. Definitely not a necessary change